README.md

```diff
@@ -1,4 +1,4 @@
-# `terraform-provider-harbor`
+# terraform-provider-harbor
```

Comment:
I googled `terraform provider harbor`, `terraform-provider-harbor`, `terraform-provider-harbor github`: this repo didn't show up. I went through the results until the last page, it's like this project isn't indexed. Hopefully this will help.